demo/demo.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'example-app',
+  template: `
+    <div class="examples">
+      <example-a></example-a>
+    
+      <example-b></example-b>
+    
+      <another-example></another-example>
+    
+      <such-example></such-example>
+    
+      <very-example></very-example>
+    
+      <much-example></much-example>
+    
+      <wow-example></wow-example>
```

Comment:
Look at the example [names](http://sd.keepcalm-o-matic.co.uk/i/such-doge-much-wow.png)

Not saying it should be changed, just funny